### PR TITLE
Fix for segues tied to gesture recognizers (and possibly other cases)

### PR DESCRIPTION
**Pull request by jai**
Tested and seems to work - please do test it yourself too, and see comment in the issue about storyboard structure (scene <-> viewcontroller relationships, if they're 1-1 or not)

### CODE REVIEW

File: natalie.swift [hunk 1]
```diff
@@ -773,11 +773,11 @@ class StoryboardFile {
     }
 
     func processStoryboard() {
-        if let xml = self.xml, viewControllers = searchAll(xml, "sceneMemberID", "viewController") {
-            for viewController in viewControllers {
+        if let xml = self.xml, viewControllers = searchAll(xml, "sceneMemberID", "viewController"), scenes = searchAll(xml, "sceneID") {
```
Comment (krzyzanowskim) [inline natalie.swift]: is this `viewControllers = searchAll(xml, "sceneMemberID", "viewController")` still necessary ?

Comment (jai) [inline natalie.swift]: Probably not...I'll double-check and update the PR

Comment (jai) [inline natalie.swift]: done

**Comment by jai** [comment]
@krzyzanowskim see d4093f5891cca66c098ffd5171df0a3ac46befd7

**Comment by jai** [comment]
Updated

**Pull request merged**